install_config/install/advanced_install.adoc

```diff
@@ -426,6 +426,203 @@ openshift_examples_modify_imagestreams=true
 |Set to `true` if pointing to a registry other than the default. Modifies the image stream location to the value of `*oreg_url*`.
 |===
 
+[[advanced-install-configuring-integrated-glusterfs-backed-registry]]
+=== Configuring an Integrated GlusterFS Backed Registry
+
+An integrated GlusterFS Registry can be configured and deployed during the initial installation
+of the cluster to offer a more reliable image registry. link:https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.1/html/container-native_storage_for_openshift_container_platform_3.4/[Red Hat Container Native Storage (CNS)] has been integrated into the {product-title} Advanced Installation procedure utilizing containerized GlusterFS storage pods to back the Docker registry.
```

Comment:
@jarrpa - this doc is in the advanced installation section of the admin guide, maybe I don't need to reference Advanced Installation since that is where the user is currently looking?